test/compeon/token/decoder_test.rb

```diff
@@ -219,4 +219,14 @@ def test_with_an_invalid_iat_claim
       ).decode
     end
   end
+
+  def test_with_a_sid_claim
+    encoded_token = JWT.encode({ attr: 'Ein Attribut', knd: 'test', sid: '123456' }, PRIVATE_KEY, 'RS256')
+
+    Compeon::Token::Decoder.new(
+      encoded_token: encoded_token,
+      key: PRIVATE_KEY.public_key,
+      token_klass: TestToken
+    ).decode
+  end
```

Comment:
This test doesn't have any assertions, is it meant as smoke test?

Reply:
Yes, as there is no `refute_raises` this is how to assure no exception is raise.